core/templates/domain/skill/skill-backend-api.service.spec.ts

```diff
@@ -254,6 +254,60 @@ describe('Skill backend API service', () => {
       expect(failHandler).toHaveBeenCalledWith('Some error in the backend.');
     }));
 
+  it(
+    'should make a request to check if skill assigned for diagnostic test.',
+    fakeAsync(() => {
+      const backendResponse = {
+        topic_names: [],
+      };
+      const successHandler = jasmine.createSpy('success');
+      const failHandler = jasmine.createSpy('fail');
+      const skillId = 'skill1';
+
+      skillBackendApiService
+        .getTopicNamesWithGivenSkillAssignedForDiagnosticTest(
+          skillId).then(successHandler, failHandler);
+
+      let req = httpTestingController.expectOne(
+        '/diagnostic_test_skill_assignment_handler/skill1'
+      );
+      expect(req.request.method).toEqual('GET');
+      req.flush(backendResponse);
+
+      flushMicrotasks();
+
+      expect(successHandler).toHaveBeenCalled();
+      expect(failHandler).not.toHaveBeenCalledWith();
```

Comment:
Should this just be not.toHaveBeenCalled()?

Reply:
Thank you for the suggestion, I have updated.